kopeme-junit5/src/main/java/de/dagere/kopeme/junit/rule/KoPeMeBasicStatement5.java

```diff
@@ -62,7 +60,7 @@ public KoPeMeBasicStatement5(final TestRunnable runnables, final Method method,
 
       if ("EXTENDED".equals(annotation.dataCollectors())) {
          datacollectors = DataCollectorList.EXTENDED;
-      } else if ("STANDARD".equals(annotation.dataCollectors()) || annotation == null) {

```

Comment:
The null check should still be there, why did you remove it?

Reply:
This was reported by IntelliJ. If annotation would be null, you would get errors everywhere where annotation.dataCollectors() is called earlier, wouldn't you?